otx/cli/tools/eval.py

```diff
@@ -136,9 +137,8 @@ def main():
     assert resultset.performance is not None
     print(resultset.performance)
 
-    if not args.save_performance:
-        args.save_performance = str(Path(args.load_weights).parent / "performance.json")
-    with open(args.save_performance, "w", encoding="UTF-8") as write_file:
+    output_path = Path(args.output) if args.output else config_manager.output_path
```

Comment:
how about just using `config_manager.output_path`? Same algorithm exists there.

Reply:
I fixed this. https://github.com/openvinotoolkit/training_extensions/pull/1946/commits/2cb8133579f0f6a36f78f4a32ef2a5287d1008e8